### PR TITLE
Use GVAR Macro Where Applicable

### PR DESCRIPTION
**Pull request by 654wak654**
**When merged this pull request will:**
- Replace the use of `ace_module_` in some places with the `GVAR` macro and it's cousins.
- As a side effect I found and fixed some functions' headers.

### CODE REVIEW

File: addons/concertina_wire/functions/fnc_handleKilled.sqf [hunk 1]
```diff
@@ -19,7 +19,7 @@ TRACE_2("params",_wire,_killer);
 private ["_distance", "_vehicle"];
 
 if (isNull _killer) then {
-    _killer = _wire getVariable ["ace_concertina_wire_lastDamager", objNull];
+    _killer = _wire getVariable [QGVAR(lastDamager), objNull];
     if (isNull _killer) then {
         private _midPoint = ((_wire selectionPosition "start") vectorAdd (_wire selectionPosition "deploy")) vectorMultiply 0.5;
         {
```
Comment (Glowbal) [inline addons/concertina_wire/functions/fnc_handleKilled.sqf]: N

Comment (654wak654) [inline addons/concertina_wire/functions/fnc_handleKilled.sqf]: N?

**Comment by esteldunedain** [comment]
I went through all the lines and could not find any error. A double check would be nice though.

File: addons/concertina_wire/functions/fnc_handleKilled.sqf (hunk 1 above)
Comment (Glowbal) [inline addons/concertina_wire/functions/fnc_handleKilled.sqf]: I was looking at this with my phone and accidentally posted that. Didn't let me delete it on my phone.

Comment (commy2) [inline addons/concertina_wire/functions/fnc_handleKilled.sqf]: The correct answer was:
O

File: addons/overheating/CfgVehicles.hpp [hunk 2]
```diff
@@ -52,7 +52,7 @@ class CfgVehicles {
                 };
                 class GVAR(CheckTemperature) {
                     displayName = CSTRING(CheckTemperatureShort);
-                    condition = "ace_overheating_enabled && {switch (currentWeapon _target) do {case (''): {false}; case (primaryWeapon _target); case (handgunWeapon _target): {true}; default {false}}}";
+                    condition = QUOTE( GVAR(enabled) && {switch (currentWeapon _target) do {case (''): {false}; case (primaryWeapon _target); case (handgunWeapon _target): {true}; default {false}}} );
```
Comment (commy2) [inline addons/overheating/CfgVehicles.hpp]: white space

File: addons/overheating/CfgVehicles.hpp [hunk 3]
```diff
@@ -23,7 +23,7 @@ class CfgVehicles {
                 };
                 class GVAR(CheckTemperature) {
                     displayName = CSTRING(CheckTemperatureShort);
-                    condition = "ace_overheating_enabled && {switch (currentWeapon _player) do {case (''): {false}; case (primaryWeapon _player); case (handgunWeapon _player): {true}; default {false}}}";
+                    condition = QUOTE( GVAR(enabled) && {switch (currentWeapon _player) do {case (''): {false}; case (primaryWeapon _player); case (handgunWeapon _player): {true}; default {false}}} );
```
Comment (commy2) [inline addons/overheating/CfgVehicles.hpp]: white space

File: addons/huntir/Dialog.hpp [hunk 4]
```diff
@@ -138,7 +138,7 @@ class ace_huntir_cam_dialog {
         x = __X + (2*0.8); y = __Y + 0.045;
         w = 0.05; h = 0.02;
         text = CSTRING(HELP);
-        action = "createDialog 'ace_huntir_help_dialog'";
+        action = QUOTE(createDialog QQGVAR(help_dialog));
```
Comment (commy2) [inline addons/huntir/Dialog.hpp]: FYI:
you can also write:
`action = QUOTE(createDialog 'GVAR(help_dialog)');`

Macros are resolved inside single quote strings.

**Comment by commy2** [comment]
looks good

File: addons/huntir/Dialog.hpp (hunk 4 above)
Comment (jonpas) [inline addons/huntir/Dialog.hpp]: That why those macros exist though, nicer highlighting. :)

**Pull request merged**